src/ckeditor.jsx

```diff
@@ -55,7 +55,7 @@ class CKEditor extends React.Component {
 		const { config, readOnly, type, onBeforeLoad, style, data } = this.props;
 		config.readOnly = readOnly;
 
-		getEditorNamespace( CKEditor.editorUrl ).then( CKEDITOR => {
+		getEditorNamespace( CKEditor.editorUrl, this.props.onNamespaceLoaded ).then( CKEDITOR => {
```

Comment:
Since we are already destructing `props` at the beginning of the method (55), maybe let's stick to it and destruct `onNamespaceLoaded` also?